manager/src/main/java/com/redhat/service/smartevents/manager/metrics/ManagedResourceOperationMapper.java

```diff
@@ -0,0 +1,65 @@
+package com.redhat.service.smartevents.manager.metrics;
+
+import java.util.Objects;
+
+import com.redhat.service.smartevents.infra.metrics.MetricsOperation;
+import com.redhat.service.smartevents.infra.models.dto.ManagedResourceStatus;
+import com.redhat.service.smartevents.infra.models.dto.ManagedResourceStatusUpdateDTO;
+import com.redhat.service.smartevents.manager.models.ManagedResource;
+
+public final class ManagedResourceOperationMapper {
```

Comment:
We used to have some (IMO, horrible) code in both `BridgeService` and `ProcessorService` that decided what _operation_ had been performed and therefore what _action_ to complete. I moved it to this new class and refactored it. IMO it is now nicer.. but this is obviously very subjective :smile: